build.sbt

```diff
@@ -373,6 +358,18 @@ lazy val releaseSettings = Seq(
 
 // Looks like some of release settings should be set for the root project as well.
 publishArtifact := false  // Don't release the root project
-publish := {}
+publish / skip := true
```

Comment:
Is this line the fix for cross release issue?

Reply:
No. The fix was to bring the `releaseSettings` from inside `releaseSettings` (which was used per sub-project) into the root project